shell/platform/darwin/ios/framework/Headers/FlutterCodecs.h

```diff
@@ -0,0 +1,83 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_FLUTTERCODECS_H_
+#define FLUTTER_FLUTTERCODECS_H_
+
+#import <Foundation/Foundation.h>
+#include "FlutterMacros.h"
+
+FLUTTER_EXPORT
+@protocol FlutterMessageCodec
```

Comment:
I have not done this yet. Question is if it makes sense to postpone that refactoring to unblock people waiting for platform/flutter channels.